api/src/common/components.py

```diff
@@ -24,6 +24,16 @@
         "alt_name": "Mercury",
         "molecular_weight": 200.59,
     },
+    "6": {
+        "chemical_formula": "MEG",
+        "alt_name": "ethane-1-2-diol",
+        "molecular_weight": 62.068,
+    },
+    "31": {
+        "chemical_formula": "TEG",
+        "alt_name": "2-[2-(2-Hydroxyethoxy)ethoxy]ethanol",
```

Comment:
Also for TEG the formula and alt_name has been swapped.
And the formula seems to be "C6H14O4"